test/contrib/sidekiq/client_tracer_test.rb

```diff
@@ -56,4 +61,9 @@ def test_empty_parentless
     assert_equal(0, span.status)
     assert_nil(span.parent)
   end
+
+  def test_delayed_extensions
```

Comment:
Would be great to convert these to RSpec, but its not a blocker for this task (out of scope.)

Reply:
Agreed!